pkg/kubelet/kubelet.go

```diff
@@ -2366,6 +2371,9 @@ func (kl *Kubelet) setNodeStatus(node *api.Node) error {
 		if !networkConfigured {
 			messages = append(reasons, "network not configured correctly")
 		}
+		if !selinuxConfigured {
+			messages = append(reasons, "SELinux is required by the cluster but not enabled on this node")
```

Comment:
Sounds like we imply here if API server is enabled with selinux-integration, all nodes must be selinux-integration enabled. Heterogeneous cluster in this case is not allowed. Is that right?